tools/quickstart/quickstart/bridge.py

```diff
@@ -57,6 +59,21 @@ def setup_interactively(base_dir, bridge_config_file) -> None:
         Path(user_file).touch()
         return
 
+    choice = click.prompt(
+        fill(
+            "The bridge client requires access to a main chain node."
+            "Do you want to run a main chain node (1) or provide a JSON RPC url (2)?"
+        )
+        + "\n",
+        type=click.Choice(("1", "2")),
+        show_choices=False,
+    )
+    if choice == "2":
+        url = read_json_rpc_url()
+        change_json_rpc_url(bridge_config_file, url)
+    elif choice != "1":
```

Comment:
what is this?

Reply:
I guess safety check, I took it from: https://github.com/trustlines-protocol/blockchain/blob/master/tools/quickstart/quickstart/validator_account.py#L72
I don't mind removing it.

Reply:
Either then write it like that where you took it from or remove it. 
The reason there was to list all possible values and have an else to catch if you missed one.